### PR TITLE
polish le gamemode ui stuff

### PR DESCRIPTION
**Pull request by toberge**
Now with descriptions:

image

Also makes tab group selection wrap around. Cuz why wouldn't it ┗( T﹏T )┛
Also also prevented the selection disappearance on the resolution dropdown. Still some odd hijinks where all those dropdowns become partially shaded for some reason.

Please gib feedback on the descriptions, the one for 30 chips is the only one I'm moderately satisfied with.

Closes part of #742 ~~(not the scoreboard things (yet))~~ also the scoreboard things:

- LEADER for players with max kills in SIX ROUNDS
- CAN WIN for players with 30 chips in FIRST TO 30 CHIPS
- Pickups reward for chips collected 
- Round X on all gamemodes

### CODE REVIEW

File: Assets/Scripts/Scoreboard/Scoreboard.cs [hunk 1]
```diff
@@ -184,6 +188,9 @@ private IEnumerator AnimateScoreProgress()
 
         int previousScore = score - scoreGain;
 
+
+
+
```
Comment (Fueredoriku) [inline Assets/Scripts/Scoreboard/Scoreboard.cs]: Why all the whitespace? 🤔

**Review by Fueredoriku — APPROVED** [review]
Looks good! 🤠

Haven't been able to test very thoroughly in game as my laptop is currently dying a slow and painful death, but from what I have been able to see this is very nice 👍

File: Assets/Scripts/Scoreboard/Scoreboard.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/Scoreboard/Scoreboard.cs]: Moved some code, may clean up later

Comment (toberge) [reply Assets/Scripts/Scoreboard/Scoreboard.cs]: Moved some code, may clean up later

Comment (toberge) [reply Assets/Scripts/Scoreboard/Scoreboard.cs]: Cleaned up and fixed a bug with the "round x/y" text 👍

**Pull request merged**